cps/templates/author.html

```diff
@@ -0,0 +1,65 @@
+{% extends "layout.html" %}
+{% block body %}
+<h2>{{title}}</h2>
+
+{% if author is not none %}
+<section class="author-bio">
+  {%if author['image_url'] is not none %}
+  <img src="{{author.image_url}}" alt="{{author.name}}" class="author-photo pull-left">
```

Comment:
Weird that Jinja2 doesn't take care of that already. Don't book titles also have a problem? If not, do you know why not?